### PR TITLE
log: add timestamp do gelf message

### PR DESCRIPTION
**Pull request by vierno**
Related to the issue #29 , this code change will add a timestamp to the log message, with decimal places to increase the precision.

### CODE REVIEW

File: log/gelfforwarder.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func (b *gelfBackend) sendMessage(parts *rawLogParts, appName, processName, cont
 			"_pid": processName,
 		},
 		RawExtra: b.extra,
+		TimeUnix: float64(time.Now().UnixNano()) / 1E9,
```
Comment (cezarsa) [inline log/gelfforwarder.go]: Nitpick: Could you please change `1E9` to `time.Second` to make it clearer when reading the code in the future?

Comment (vierno) [reply log/gelfforwarder.go]: Done! =)

**Comment by guilhermebr** [comment]
@vierno please do a rebase with master to fix the goimport errors

**Comment by vierno** [comment]
@guilhermebr Done!

**Review by guilhermebr — APPROVED** [review]

**Pull request merged**